data/json/items/gun/308.json

```diff
@@ -446,5 +455,51 @@
     "min_cycle_recoil": 2700,
     "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "ar10mag_20rd", "ar10_makeshiftmag" ] } ],
     "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rfb_308",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "bullpup rifle 7.62x51" },
+    "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the Cold War battle rifle and a 7.62x51mm cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rfb_308",
+        "name": { "str_sp": "Kel-Tec RFB" },
+        "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using magazines of the FN FAL and a .308 NATO cartridge.  The main design features include forward case ejection, duplication of magazine resease button and safety trigger for left-handers and a posted barrel for increased accuracy.",
+        "weight": 1
+      }
+    ],
+    "weight": "3900 g",
+    "volume": "3212 ml",
+    "longest_side": "701 mm",
+    "barrel_length": "470 mm",
+    "price": 180000,
+    "price_postapoc": 4000,
+    "to_hit": -2,
+    "material": [ "steel", "aluminum", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "308" ],
+    "dispersion": 150,
+    "durability": 6,
+    "min_cycle_recoil": 2700,
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
```

Comment:
If this gun is forward-ejecting, wouldn’t that require a different brass catcher design and the removal of this slot, as can be observed on other such guns like the F2000?

Reply:
I was thinking about it. Maybe it's wrong assumption, but i see ingame brass catcher as a "universal" model (or one that comes with a lot of duct tape), like this one 
![изображение](https://github.com/CleverRaven/Cataclysm-DDA/assets/133244791/369ef1d6-1b2f-411f-9ba1-1373d2a51527)

https://youtu.be/DDyH2q8VaK8 - i assume, that it possible to slightly bend brass catcher with a strap, to catch ejected cases, given that they fall out rather than thrown away. It's definately not a ideal solution, but _i think_ not impossible.

Reply:
Done